Sources/Pageboy/Protocols/PageboyViewControllerDelegate.swift

```diff
@@ -33,6 +33,16 @@ public protocol PageboyViewControllerDelegate: class {
                                didScrollTo position: CGPoint,
                                direction: PageboyViewController.NavigationDirection,
                                animated: Bool)
+
+    /// The page view controller did not (!) complete scroll to a new page.
+    ///
+    /// - Parameters:
+    ///   - pageboyViewController: The Page view controller.
+    ///   - index: The expected new page index, that was not (!) scrolled to.
+    ///   - previousIndex: The page index returned to.
+    func pageboyViewController(_ pageboyViewController: PageboyViewController,
+                               abortScrollToPageAt index: PageboyViewController.PageIndex,
+                               returnToPageAt previousIndex: PageboyViewController.PageIndex)
```

Comment:
Could we maybe call this `didCancelScrollToPageAt`??

Just fits in a bit better with UIKit semantics and style...

Reply:
Done.